main_test.go

```diff
@@ -21,6 +23,64 @@ type MainTest struct {
 
 func init() { RegisterTestSuite(&MainTest{}) }
 
+func (t *MainTest) TestHandleCustomEndpointWithProdGCSEndpoint() {
+	url, err := url.Parse("https://storage.googleapis.com:443")
+	AssertEq(nil, err)
```

Comment:
nit: Empty line before and after assert statement

Reply:
Ideally, we separate the arrange, act and assert block by new line. 

That assert is part of arrange block, that is why I haven't added any newline.

Adding empty line after the assert block. And adding comment to separate the `arrange`, `act` and `assert` block.

Reply:
Got it thanks